public/actions/TagActions/getTag.js

```diff
@@ -11,11 +15,12 @@ function requestTagGet() {
     };
 }
 
-function recieveTagGet(tag) {
+function recieveTagGet(tag, canEdit) {
     return {
-        type:       TAG_GET_RECEIVE,
-        tag:        tag,
-        receivedAt: Date.now()
+        type:        TAG_GET_RECEIVE,
+        tag:         tag,
+        tagEditable: canEdit,
```

Comment:
Yeah, it's passed into the reducer and makes sure all the UI components are usable or disabled depending on permissions settings.